src/popup/utils/constants.js

```diff
@@ -65,19 +75,9 @@ export const defaultNetworks = [
     tipContractV2: 'ct_2ZEoCKcqXkbz2uahRrsWeaPooZs9SdCv6pmC4kc55rD4MhqYSu',
     name: 'Testnet',
   },
-  {
-    url: 'https://mainnet.aeternity.io',
-    networkId: 'ae_mainnet',
-    middlewareUrl: 'https://mainnet.aeternity.io/mdw',
-    explorerUrl: 'https://explorer.aeternity.io',
-    compilerUrl: 'https://compiler.aepps.com',
-    backendUrl: 'https://raendom-backend.z52da5wt.xyz',
-    tipContractV1: 'ct_2AfnEfCSZCTEkxL5Yoi4Yfq6fF7YapHRaFKDJK3THMXMBspp5z',
-    name: 'Mainnet',
-  },
 ];
 
-export const defaultNetwork = process.env.NETWORK === 'Testnet' ? defaultNetworks[0] : defaultNetworks[1];
+export const defaultNetwork = process.env.NETWORK === 'Testnet' ? defaultNetworks[1] : defaultNetworks[0];
```

Comment:
Rather subjective comment - skip it if you think it's a bullshit:
I red once a nice rule for making this kind of one liners to be more easy to follow with eyes:
```
foo = (condition)
  ? result1
  : result2;
```
It's easier to read because it mimics the lists done in text editors.